addons/editor-colored-context-menus/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Colored context menus",
-  "description": "Makes block right click context menus colorful.",
+  "description": "Makes the background of block right click context menus colorful.",
```

Comment:
If the "black" preset for "customizable block colors" is selected, this addon changes the border and text color, not the background.

Reply:
Should I revert it, then?